src/components/tile/tile.component.js

```diff
@@ -24,33 +19,35 @@ const Tile = (props) => {
       <TileContent
         key={ key }
         width={ contentWidth }
+        isHorizontal={ isHorizontal(orientation) }
+        isVertical={ isVertical(orientation) }
+        pt={ isVertical(orientation) && (props.pt, props.py, p) }
```

Comment:
This will pass in a boolean sometimes, and will never set anything but the `p` value, something like this will work

```suggestion
        { ...(isVertical(orientation) && {
          pt: props.pt || props.py || p,
          pb: props.pb || props.py || p
        }) }
        { ...(isHorizontal(orientation) && {
          pr: props.pr || props.px || p,
          pl: props.pl || props.px || p
        }) }
```